ui/dashboards/src/components/TimeRangeControls/TimeRangeControls.tsx

```diff
@@ -29,7 +31,7 @@ export const TIME_OPTIONS: TimeOption[] = [
 ];
 
 export function TimeRangeControls() {
-  const { timeRange, setTimeRange } = useDashboardTimeRange();
```

Comment:
removed useDashboardTimeRange so we only have one hook `useTimeRange` :)

Reply:
Yeah I definitely prefer having one hook, it simplifies the implementation a ton! We talked about this offline but for context, I added this internal version to not expose extra properties in the plugin runtime. 

@saminzadeh @LukeTillman previously I got feedback from you both / this [discussion](https://github.com/perses/perses/discussions/91#discussioncomment-1425254) that plugin system should ideally only expose absolute time, thoughts on adding the [separate absoluteTimeRange](https://github.com/perses/perses/blob/c8e8239939a175516dc03d6a4c4fcb80895762b6/ui/plugin-system/src/runtime/TimeRangeProvider/TimeRangeProvider.tsx#L24-L30) back to plugin-system? I think it's fine since TimeRangeControls also needs to use the hook and needs access to both since you can’t really derive relative time from absolute time.